.devcontainer/devcontainer.json

```diff
@@ -1,83 +1,65 @@
-  "extensions": [
-    // https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments
-    "aaron-bond.better-comments",
-    // https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense
-    "christian-kohler.npm-intellisense",
-    // https://marketplace.visualstudio.com/items?itemName=christian-kohler.path-intellisense
-    "christian-kohler.path-intellisense",
-    // https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
-    "davidanson.vscode-markdownlint",
-    // https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
-    "dbaeumer.vscode-eslint",
-    // https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens
-    "eamodio.gitlens",
-    // https://marketplace.visualstudio.com/items?itemName=editorconfig.editorconfig
-    "editorconfig.editorconfig",
-    // https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
-    "esbenp.prettier-vscode",
-    // https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree
-    "Gruntfuggly.todo-tree",
-    // https://marketplace.visualstudio.com/items?itemName=ionutvmi.path-autocomplete
-    "ionutvmi.path-autocomplete",
-    // https://marketplace.visualstudio.com/items?itemName=KnisterPeter.vscode-commitizen
-    "knisterpeter.vscode-commitizen",
-    // https://marketplace.visualstudio.com/items?itemName=lunaryorn.fish-ide
-    "lunaryorn.fish-ide",
-    // https://marketplace.visualstudio.com/items?itemName=mrmlnc.vscode-json5
-    "mrmlnc.vscode-json5",
-    // https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker
-    "ms-azuretools.vscode-docker",
-    // https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml
-    "redhat.vscode-yaml",
-    // https://marketplace.visualstudio.com/items?itemName=sissel.shopify-liquid
-    "sissel.shopify-liquid",
-    // https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
-    "streetsidesoftware.code-spell-checker",
-    // https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml
-    "tamasfe.even-better-toml",
-    // https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter
-    "travisthetechie.write-good-linter",
-    // https://marketplace.visualstudio.com/items?itemName=travisthetechie.skyapps.fish-vscode
-    "skyapps.fish-vscode",
-    // https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
-    "yzhang.markdown-all-in-one"
-  ],
+      // Add the IDs of extensions you want installed when the container is created.
+      "extensions": ["GitHub.vscode-pull-request-github"]
+    }
+  },
 
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [
-    // Jekyll server
-    4000,
-    // Live reload server
-    35729,
-    // OpenSSH server
-    2222
-  ],
+  "remoteUser": "codespace",
 
```

Comment:
All of the above is necessary for commit signing and should not have been removed.

I may have to fix this instead of merging as-is.